packages/taquito-rpc/src/types.ts

```diff
@@ -603,6 +603,18 @@ export type OperationContents =
   | OperationContentsSmartRollupRecoverBond
   | OperationContentsSmartRollupTimeout;
 
+export type OperationContentsWithFee =
```

Comment:
Didn't see this used is this only defined and exported for end user?

Reply:
you're right, it should be removed instead. good catch